fabtests/scripts/runfabtests.sh

```diff
@@ -278,21 +281,54 @@ function read_exclude_file {
 		# Ignore patterns that are comments or just whitespaces
 		ignore_pattern="#.*|^[\t ]*$"
 		if [[ ! "$pattern" =~ $ignore_pattern ]]; then
-			if [ -z "$EXCLUDE" ]; then
-				EXCLUDE="$pattern"
+			if [ -z "$file_excludes" ]; then
+				file_excludes="$pattern"
 			else
-				EXCLUDE="${EXCLUDE},$pattern"
+				file_excludes="${file_excludes},$pattern"
 			fi
 		fi
-	done < "$exclude_file"
+	done < "$excl_file"
+}
+
+function auto_exclude {
+	local excl_file
+
+	excl_file="./fabtests/test_configs/${PROV}/${PROV}.exclude"
+	if [[ ! -f "$excl_file" ]]; then
+		excl_file="./test_configs/${PROV}/${PROV}.exclude"
+		if [[ ! -f "$excl_file" ]]; then
+			excl_file="../test_configs/${PROV}/${PROV}.exclude"
+			if [[ ! -f "$excl_file" ]]; then
+				return
+			fi
+		fi
+	fi
+
+	read_exclude_file ${excl_file}
+	cur_excludes=${file_excludes}
+	file_excludes=""
+}
+
+function set_excludes {
+	if [[ -n "$input_excludes" ]]; then
+		cur_excludes=${input_excludes}
+		return
+	fi
+
+	if [[ -n "$file_excludes" ]]; then
+		cur_excludes=${file_excludes}
+		return
+	fi
```

Comment:
I think if we append the `file_excludes` to `cur_excludes` we can maintain the behavior of accepting both exclude file and exclude list on the command line.

Reply:
done